.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+# Purpose
+
+**Type of PR**
+* [ ] Bug Fix
+* [ ] Feature Work
+* [ ] Revert
+* [ ] Refactor / Code Cleanup
+* [ ] Other (fill in...)
+
+**Review Urgency**
+* [ ] ASAP
+* [ ] Next 2 hours
+* [ ] End of Day
+* [ ] Tomorrow
+* [ ] End of Week
+* [ ] Other (fill in...)
+
+
+## Short Description
+<!-- Feature / Problem overview -->
+
+
+## Changelog / Code Overview
+<!-- What was changed / added / removed and why. If you changed the frontend please include screenshots -->
+
+
+## Test Plan
+<!-- How was this code tested / How should reviewers test it? -->
+
+
+## Deploy Plan
+<!--
+How will this be deployed? Are there any special cases you need to think about?
+How will this change be reverted if there is an issue?
+-->
+
+
+## What to look for
+<!-- Call out what each team/reviewer should look for and a rough time estimate. -->
+
+
+## Additional Details
```

Comment:
I'd like this to just be titled *Related Issue/Ticket* so that its totally clear what to put here. This is going to be an open source repo so I'm trying to remove any of the vagueness of the original template!